ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-nightly.yaml

```diff
@@ -633,6 +633,19 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: rosa-nightly-sts
+- as: e2e-osd-ccs-aws
```

Comment:
Could you explain a little bit about the reasons for choosing 4.11, rather than 4.12 ?

Reply:
In the past, the latest nightly version on the ocm side is 4.11. Even now we have had the 4.12 nightly versions, the latest stable version is 4.11, we can not make sure if 4.12 OCP is ready for OSD. To validate the provision process, I use 4.11 to make sure the steps go smoothly. In the future, after I have a discussion with yufchang, we will decide whether use 4.12 instead of 4.11.